ext/oj/parser.c

```diff
@@ -1263,7 +1263,7 @@ static VALUE parser_new(int argc, VALUE *argv, VALUE self) {
 // from this function. A delegate must be added before the parser can be
 // used. Optionally oj_parser_set_options can be called if the options are not
 // set directly.
-VALUE oj_parser_new() {
```

Comment:
# JSON Quickstart

Reply:
This is a closed MR. What are you trying to achieve?